packages/@glimmer/syntax/lib/traversal/walker.ts

```diff
@@ -33,40 +33,21 @@ export default class Walker {
     switch (node.type) {
       case 'Block':
       case 'Template':
-        return visitors.Program(this, node as unknown as ASTv1.Program, callback);
+        walkBody(this, node.body, callback);
+        return;
       case 'ElementNode':
-        return visitors.ElementNode(this, node, callback);
+        walkBody(this, node.children, callback);
+        return;
       case 'BlockStatement':
```

Comment:
TIL:
```hbs
{{#if a}}

  hi
{{else if b}}
  hello
{{else}}
  hola
{{/if}}
```

everything that's not the _first_ block is in "inverse"
![image](https://github.com/glimmerjs/glimmer-vm/assets/199018/48e240f6-d4cb-4e1c-b13e-032b9a8a0b23)

kinda good :see_no_evil: